src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindowBase.cs

```diff
@@ -1,358 +1,358 @@
+                if (deviceInfo == null || deviceInfo.NetworkManager == null || !deviceInfo.NetworkManager.IsConnected || spatialCoordinateSystemParticipant == null)
+                {
+                    sharedSpatialCoordinateStatusMessage = notConnectedMessage;
+                }
+                else if (!spatialCoordinateSystemParticipant.PeerDeviceHasTracking)
+                {
+                    sharedSpatialCoordinateStatusMessage = trackingLostStatusMessage;
+                }
+                else if (deviceInfo.IsTrackingStalled)
+                {
+                    sharedSpatialCoordinateStatusMessage = trackingStalledStatusMessage;
+                }
+                else if (spatialCoordinateSystemParticipant.PeerIsLocatingSpatialCoordinate)
+                {
+                    sharedSpatialCoordinateStatusMessage = locatingSharedSpatialCoordinate;
+                }
+                else if (!spatialCoordinateSystemParticipant.PeerSpatialCoordinateIsLocated)
+                {
+                    sharedSpatialCoordinateStatusMessage = notLocatedSharedSpatialCoordinate;
+                }
+                else
+                {
+                    sharedSpatialCoordinateStatusMessage = locatedSharedSpatialCoordinateMessage;
+                }
+
+                GUILayout.Label("Shared spatial coordinate status", boldLabelStyle);
+                GUILayout.Label(sharedSpatialCoordinateStatusMessage);
+
+                string calibrationStatusMessage;
+                if (compositionManager != null && compositionManager.IsCalibrationDataLoaded)
+                {
+                    calibrationStatusMessage = calibrationLoadedMessage;
+                }
+                else
+                {
+                    calibrationStatusMessage = calibrationNotLoadedMessage;
+                }
+
+                EditorGUILayout.Space();
+                if (showCalibrationStatus)
+                {
+                    GUILayout.Label("Calibration status", boldLabelStyle);
+                    GUILayout.Label(calibrationStatusMessage);
+                }
+                else
+                {
+                    // Empty label to remain as a placeholder for the Calibration status
+                    GUILayout.Label(string.Empty, boldLabelStyle);
+                    GUILayout.Label(string.Empty);
+                }
+
+                EditorGUILayout.Space();
+
+                if (GUILayout.Button(new GUIContent("Locate Shared Spatial Coordinate", "Detects the shared location used to position objects in the same physical location on multiple devices")))
+                {
+                    CompositorWorldAnchorLocalizationManager.Instance.RunRemoteLocalizationWithWorldAnchorPersistence(spatialCoordinateSystemParticipant, ArUcoMarkerDetectorSpatialLocalizer.Id, new MarkerDetectorLocalizationSettings
+                    {
+                        MarkerID = 0,
+                        MarkerSize = 0.067f,

```

Comment:
>MarkerSize = 0.067f, [](start = 24, length = 20)

Can we leave this as 0.1f for now until I've finished adding the UI for choosing this value?